css/date_picker.styl

```diff
@@ -97,10 +95,13 @@ calendarWidth = 250px
     background #ffffff
     position relative
     border-radius 50%
+    margin 0
```

Comment:
Needed for `button` element because currently we are not importing any `reset.css`